eva_submission/eload_ingestion.py

```diff
@@ -314,6 +320,41 @@ def run_variant_load_workflow(self):
             raise e
         return output_dir
 
+    def insert_browsable_files(self):
+        with self.get_pg_conn() as conn:
+            # insert into browsable file table
+            insert_query = "insert into browsable_file (file_id,ena_submission_file_id,filename,project_accession,assembly_set_id) " \
+                           "select file.file_id,ena_submission_file_id,filename,project_accession,assembly_set_id " \
+                           "from (select * from analysis_file af " \
+                           "join analysis a on a.analysis_accession = af.analysis_accession " \
+                           "join project_analysis pa on af.analysis_accession = pa.analysis_accession " \
+                           f"where pa.project_accession = '{self.project_accession}' ) myfiles " \
+                           "join file on file.file_id = myfiles.file_id where file.file_type ilike 'vcf';"
+            execute_query(conn, insert_query)
```

Comment:
For the sake of this process being idempotent, it would be great if this insertion is done only if the entry isn't already there.